universal-login-relayer/knexfile.js

```diff
@@ -0,0 +1,45 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'postgresql',
+    connection: {
+      database: 'universal_login_relayer_development',
+      user:     'postgres',
+      password: 'postgres'
+    },
+    migrations: {
+      tableName: 'knex_migrations'
+    }
+  },
+
+  test: {
+    client: 'postgresql',
+    connection: {
+      database: 'universal_login_relayer_test',
+      user:     'postgres',
+      password: ''
+    },
+    migrations: {
+      tableName: 'knex_migrations'
+    }
+  },
+
+  production: {
+    client: 'postgresql',
+    connection: {
+      database: 'universal_login_relayer_production',
+      user:     'username',
```

Comment:
Should be taken from ENV and we should make sure it will work after deployment to heroku.

Reply:
✅